modules/accountplan/view/hovedbok.php

```diff
@@ -285,45 +322,54 @@
     </td>
   </tr>
   <tr>
+    <? // Freetext match ?>
     <td class="menu">Fritekst match</td>
     <td colspan="2"><input type="text" name="accountplan.AccountLineFreeTextMatch" value="<? print $account->AccountLineFreeTextMatch  ?>" size="30"> Brukes av bankavstemming/kontoutskrift importen for &aring; automatisk sette dette kontonummeret p&aring; en transaksjon som har en matchende tekst</td>
   </tr>
   <tr class="result">
+    <? // Log ?>
     <th colspan="5">Logg</th>
   </tr>
   <tr>
+    <? // Last used in journal ?>
     <td class="menu">Sist brukt i bilag</td>
     <td></td>
     <td><? print $_lib['format']->Date($account->LastUsedTime) ?></td>
     <td></td>
     <td></td>
   </tr>
   <tr>
+    <? // Created by ?>
     <td class="menu">Opprettet av </td>
     <td></td>
     <td><? print $_lib['format']->PersonIDToName($account->InsertedByPersonID)  ?></td>
     <td></td>
     <td><? print $_lib['format']->Date($account->InsertedDateTime)  ?></td>
   </tr>
   <tr>
+    <? // Changed by by ?>
     <td class="menu">Endret av </td>
     <td></td>
     <td><? print $_lib['format']->PersonIDToName($account->UpdatedByPersonID)  ?></td>
     <td></td>
     <td><? print $_lib['format']->Date($account->TS)  ?></td>
   </tr>
   <tr>
+    <? // Save ?>
     <td colspan="2" align="right"><? if($_lib['sess']->get_person('AccessLevel') >= 2) { print $_lib['form3']->submit(array('value'=>'Lagre (S)', 'name'=>'action_accountplan_update', 'accesskey'=>'S')); } ?></td>
     </form>
   </tr>
   <? if($_lib['sess']->get_person('AccessLevel') >= 2) { ?>
   <tr>
     <td colspan="2" align="right">
         <form name="delete" action="<? print $_SETUP['DISPATCH'] ?>t=accountplan.list&accountplan_type=hovedbok" method="post">
-        <? print $_lib['form3']->hidden(array('name'=>'AccountPlanID', 'value'=>$AccountPlanID)) ?>
-        <? print $_lib['form3']->submit(array('value'=>'Deaktiver (D)', 'name'=>'action_accountplan_deactivate', 'accesskey'=>'D')) ?>
-        <? if($_lib['sess']->get_person('AccessLevel') > 3) {
-            print $_lib['form3']->submit(array('value'=>'Slett (D)', 'name'=>'action_accountplan_delete', 'accesskey'=>'', 'confirm' => 'Er du sikker p&aring; at du vil slette kontoen'));
+        <? // Account plan ID
+         print $_lib['form3']->hidden(array('name'=>'AccountPlanID', 'value'=>$AccountPlanID))
+         // Deactivate
+         print $_lib['form3']->submit(array('value'=>'Deaktiver (D)', 'name'=>'action_accountplan_deactivate', 'accesskey'=>'D'))
+         if($_lib['sess']->get_person('AccessLevel') > 3) {
+            // Are you sure you want to delete the account plan?
```

Comment:
Are you sure you want to delete the account plan? => Delete | Are you sure you want to delete the account plan?